site/docs/tasks/custom-taxonomy.md

```diff
@@ -20,15 +20,15 @@ This task describes how to deploy Fybrik with a custom `taxonomy.json` file that
 
 ## Taxonomy Compile CLI tool 
 
-A CLI tool for compiling a base taxonomy and zero or more taxonomy layers is provided in our repo.
+A CLI tool for compiling a base taxonomy and zero or more taxonomy layers is provided in the [taxonomy-cli repository](https://github.com/fybrik/taxonomy-cli), along with a [Docker image](https://github.com/fybrik/taxonomy-cli/pkgs/container/taxonomy-cli) to directly run the tool.
 
 The base taxonomy can be found in [`charts/fybrik/files/taxonomy/taxonomy.json`](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/taxonomy/taxonomy.json) and example layers can be found in [`samples/taxonomy/example`](https://github.com/fybrik/fybrik/tree/master/samples/taxonomy/example).
 
-The following command can be used from the root directory of our repo to run the Taxonomy Compile CLI tool. 
+The following command can be used to run the Taxonomy Compile CLI tool from the provided Docker image.
```

Comment:
I'm not sure about "**from** the provided Docker image. I'd say **by** or **with**. (you can validate with  @elsalant )

Reply:
Validated